### DISCUSSION TITLE
Regarding the Unity "build" folder size in production

### DISCUSSION THREAD

**Discussion by nomanoff** [jeffreylanters/react-unity-webgl#219] [Question and Answer] [answered]
I have successfully integrated the library on my React app, but what I am concerned is the xxxxx.data file size. 
It is too large (1.1gb) and I was confused how  am I going to deploy it to prod. 
What to do in this case? Where do I save (or keep) the "build" folder that contains all my unity project? 
Could not even push it to GitHub. I had to add the folder on .gitignore  file. 
I am new to this and could not find an answer.
Any advice would be much appreciated. 
Thank you.

**Comment by jeffreylanters** [comment, marked as answer]
Hi. This has nothing to do with this module in particular, and so won't the following answer be. There are several approaches. From uploading your build manually on your server, to serving it from some kind of CDN. But besides that, you might want to look into optimising your build. I can't image a build with a size over a couple hundred megabytes to run good in a browser environment. I hope this informs you. If you have any further questions, feel free to reply to this thread!

**Reply by nomanoff** [reply]
Thank you for your response. Does that mean this is not a good idea to run our (around 1gb) build project in the browser, right?
I will discuss using this with my team further and if we have other questions, I will try to discuss with you guys here.
Thank you.

**Reply by jeffreylanters** [reply]
Your welcome, I would highly recommend into looking to reduce your build's size. I don't know what kind of game it is, but loading all this data in memory is never a good idea in a browser. But if it is separate over for example several scenes of sections in your game, you could try to use [Streaming Assets](https://github.com/jeffreylanters/react-unity-webgl#defining-the-streaming-assets-url).

**Reply by nomanoff** [reply]
I see. Thanks for your help and time
